extensions/amp-jwplayer/0.1/amp-jwplayer.js

```diff
@@ -53,25 +65,39 @@ class AmpJWPlayer extends AMP.BaseElement {
   /** @override */
   buildCallback() {
     this.contentid_ = userAssert(
-        (this.element.getAttribute('data-playlist-id') ||
-      this.element.getAttribute('data-media-id')),
-        'Either the data-media-id or the data-playlist-id ' +
+      (this.element.getAttribute('data-playlist-id') ||
+        this.element.getAttribute('data-media-id')),
+      'Either the data-media-id or the data-playlist-id ' +
       'attributes must be specified for <amp-jwplayer> %s',
-        this.element);
+      this.element);
 
     this.playerid_ = userAssert(
-        this.element.getAttribute('data-player-id'),
-        'The data-player-id attribute is required for <amp-jwplayer> %s',
-        this.element);
+      this.element.getAttribute('data-player-id'),
+      'The data-player-id attribute is required for <amp-jwplayer> %s',
+      this.element);
+    
+    this.contentSearch_ = this.element.getAttribute('data-content-search') || false;
+    this.contentContextual_ = this.element.getAttribute('data-content-contextual') || false;
+    this.contentRecency_ = this.element.getAttribute('data-content-recency') || false;
+    this.contentBackfill_ = this.element.getAttribute('data-content-backfill') || false;
   }
 
 
   /** @override */
   layoutCallback() {
     const iframe = this.element.ownerDocument.createElement('iframe');
+    const contentSearch = this.contentSearch_ ? 'search=' + encodeURIComponent(this.contentSearch_) : ''
```

Comment:
Do we want to encode these individually? Or can we just do at the end with the other encodings? Line 99